packages/marshal/src/marshal.js

```diff
@@ -67,7 +79,6 @@ export function makeMarshal(
   };
 
   /**
-   * @template Slot
    * @type {Serialize<Slot>}
```

Comment:
Why don't you need the `@template Slot` here?

Reply:
`Slot` is already the template of the outer function,